File: Atividades em Python/Ex19 - Indicador de velocidade.py
acele = float(input("Digite a aceleração: "))
vel = float(input("Digite a velocidade inicial: "))
tempo = float(input("Digite o tempo de percurso: "))
vel_final = vel + (acele * tempo)

if vel_final <= 40:
    print("Veiculo muito lento")
elif vel_final > 40 and vel_final <= 60:
    print("Velocidade permitida")
elif vel_final > 60 and vel_final <= 80:
    print("Velocidade de cruzeiro")
elif vel_final > 80 and vel_final <= 120:
    print("Veiculo rápido")
elif vel_final > 120:
    print("Veiculo muito rápido")

File: Atividades em C/Ex19 - Indicador de velocidade.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  float a, V0, t, V;
  
  printf("\n Digite o valor da aceleracao:\n");
  scanf("%f", &a);
 
  printf("\n Digite o valor da velocidade inicial:\n ");
  scanf("%f", &V0);
 
  printf("\n Digite o valor do tempo: \n");
  scanf("%f", &t);

  V = ((V0 + (a*t)) /3.6);
     
          if (V <= 40) 
           
 {  printf("Velocidade de %.2f Km/h.Veiculo muito lento!\n",V );
            }
          else if ((V > 40) && (V <= 60))
 {  printf("Velocidade de %.2f Km/h.Velocidade permitida!\n",V); 
            }
          else if ((V > 60) && (V <= 80)) 
 {  printf("Velocidade de %.2f Km/h.Velocidade de cruzeiro!\n",V);
            }
          else if  ((V > 80) && (V <= 120))
 {  printf("Velocidade de %.2f Km/h.Veiculo rapido!\n",V);
            }
          
          else {
        printf("Velocidade de %.2f Km/h.Veiculo muito rapido!\n",V);
          }
          
  return 0;

}